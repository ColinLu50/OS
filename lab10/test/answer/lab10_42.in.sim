8583
886
874
473
870
465
