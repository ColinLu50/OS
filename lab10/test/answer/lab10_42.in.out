FCFS
399 -> 183 -> 106 -> 379 -> 281 -> 155 -> 444 -> 76 -> 227 -> 280 -> 126 -> 333 -> 67 -> 2 -> 383 -> 343 -> 166 -> 361 -> 275 -> 136 -> 238 -> 468 -> 129 -> 97 -> 55 -> 61 -> 152 -> 43 -> 159 -> 349 -> 343 -> 122 -> 25 -> 267 -> 378 -> 313 -> 163 -> 107 -> 407 -> 304 -> 193 -> 50 -> 285 -> 208 -> 154 -> 202 -> 241 -> 160 -> 266 -> 470 -> 446 -> 466 -> 475 -> 82 -> 390 -> 257 -> 157 -> 315 -> 105 -> 190 -> 266
8583
SSTF
399 -> 407 -> 390 -> 383 -> 379 -> 378 -> 361 -> 349 -> 343 -> 343 -> 333 -> 315 -> 313 -> 304 -> 285 -> 281 -> 280 -> 275 -> 267 -> 266 -> 266 -> 257 -> 241 -> 238 -> 227 -> 208 -> 202 -> 193 -> 190 -> 183 -> 166 -> 163 -> 160 -> 159 -> 157 -> 155 -> 154 -> 152 -> 136 -> 129 -> 126 -> 122 -> 107 -> 106 -> 105 -> 97 -> 82 -> 76 -> 67 -> 61 -> 55 -> 50 -> 43 -> 25 -> 2 -> 444 -> 446 -> 466 -> 468 -> 470 -> 475
886
SCAN
399 -> 390 -> 383 -> 379 -> 378 -> 361 -> 349 -> 343 -> 333 -> 315 -> 313 -> 304 -> 285 -> 281 -> 280 -> 275 -> 267 -> 266 -> 257 -> 241 -> 238 -> 227 -> 208 -> 202 -> 193 -> 190 -> 183 -> 166 -> 163 -> 160 -> 159 -> 157 -> 155 -> 154 -> 152 -> 136 -> 129 -> 126 -> 122 -> 107 -> 106 -> 105 -> 97 -> 82 -> 76 -> 67 -> 61 -> 55 -> 50 -> 43 -> 25 -> 2 -> 0 -> 407 -> 444 -> 446 -> 466 -> 468 -> 470 -> 475
874
C-SCAN
399 -> 390 -> 383 -> 379 -> 378 -> 361 -> 349 -> 343 -> 333 -> 315 -> 313 -> 304 -> 285 -> 281 -> 280 -> 275 -> 267 -> 266 -> 257 -> 241 -> 238 -> 227 -> 208 -> 202 -> 193 -> 190 -> 183 -> 166 -> 163 -> 160 -> 159 -> 157 -> 155 -> 154 -> 152 -> 136 -> 129 -> 126 -> 122 -> 107 -> 106 -> 105 -> 97 -> 82 -> 76 -> 67 -> 61 -> 55 -> 50 -> 43 -> 25 -> 2 -> 0 -> 481 -> 475 -> 470 -> 468 -> 466 -> 446 -> 444 -> 407
473
LOOK
399 -> 390 -> 383 -> 379 -> 378 -> 361 -> 349 -> 343 -> 333 -> 315 -> 313 -> 304 -> 285 -> 281 -> 280 -> 275 -> 267 -> 266 -> 257 -> 241 -> 238 -> 227 -> 208 -> 202 -> 193 -> 190 -> 183 -> 166 -> 163 -> 160 -> 159 -> 157 -> 155 -> 154 -> 152 -> 136 -> 129 -> 126 -> 122 -> 107 -> 106 -> 105 -> 97 -> 82 -> 76 -> 67 -> 61 -> 55 -> 50 -> 43 -> 25 -> 2 -> 407 -> 444 -> 446 -> 466 -> 468 -> 470 -> 475
870
C-LOOK
399 -> 390 -> 383 -> 379 -> 378 -> 361 -> 349 -> 343 -> 333 -> 315 -> 313 -> 304 -> 285 -> 281 -> 280 -> 275 -> 267 -> 266 -> 257 -> 241 -> 238 -> 227 -> 208 -> 202 -> 193 -> 190 -> 183 -> 166 -> 163 -> 160 -> 159 -> 157 -> 155 -> 154 -> 152 -> 136 -> 129 -> 126 -> 122 -> 107 -> 106 -> 105 -> 97 -> 82 -> 76 -> 67 -> 61 -> 55 -> 50 -> 43 -> 25 -> 2 -> 475 -> 470 -> 468 -> 466 -> 446 -> 444 -> 407
465
